src/styles.typ

```diff
@@ -45,6 +45,9 @@
     /// set the default to `0` and to `auto` for the mark you want to
     /// shorten the path to. Set to `none` to disable path shortening.
     shorten-to: auto,
+    /// Place marks post path transformation. This is not honored per mark, but
+    /// for all marks on a path.
```

Comment:
Whats stopping it from being a per-mark option?

Reply:
Marks on transformed paths may be bigger, path shortening gets tricky because we would need to un-transform the path first if it was transformed by a previous mark… makes things necessary complicated IMHO.

From a user perspective, it makes no sense supporting it per mark.

Reply:
Yes thats true